pynicotine/gtkgui/wishlist.py

```diff
@@ -220,15 +237,20 @@ def on_show(self, *args):
 
         text = self.searches.notebook.get_tab_label(page).full_text
 
+        if text.startswith("(Wish) "):
+            text = text[7:]
```

Comment:
- Fixed: Items starting with "(Wish) " appear in the TextEntry when opening the Wishlist popover

Reply:
There are similar issues if you search files for users and rooms. Perhaps we could exclude the prefix from the tooltip text?

Reply:
Oh I hadn't noticed that.. Getting text from the tooltip sounds fine to me, certainly better than these useless queries being bought into wishlist

I haven't a clue where to go looking for tooltip text, nor how to get it in here?

Reply:
The tooltip text/full_text is already used at the moment, but the (Wish) prefix is added to it here: https://github.com/nicotine-plus/nicotine-plus/blob/6aae4e7a0840883231541cafed1037b39a201c23/pynicotine/gtkgui/search.py#L186-L200

Applying `"(%s) %s" % (tab.mode_label, text)` for label instead of full_text should do.

Reply:
Alternatively, without using the tooltip text:

```
    def on_show(self, *args):

        page = self.searches.get_nth_page(self.searches.get_current_page())

        if page is None:
            return

        text = None

        for search in self.searches.searches.values():
            tab = search.get("tab")

            if tab is not None and tab.Main == page:
                text = tab.text
                break

        if not text:
            return

        if text in self.wishes:
            # Highlight existing wish row
```

Reply:
Thanks, I've gone for the proper way, because I see this standard method elsewhere and it is less dependent on external code, for instance if the tooltip were ever altered for some reason then this could lead to another similar bug.